app/views/glossary_terms/_form.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= form_with model: @glossary_term, html: { multipart: true } do |f| %>
+
+  <div class="form-group mt-3">
+    <%= f.label :name, :glossary_term_name.t + ":" %>
+    <%= f.text_field :name, class: "form-control" %>
+  </div>
+
+  <div class="form-group mt-3">
+    <%= f.label :description, :glossary_term_description.t + ":" %>
+    <%= f.text_area :description, rows: 20, class: "form-control" %>
+  </div>
+
+  <%# Currently, allow image only when creating term %>
+  <% if action == :new %>
```

Comment:
The home-made `action` flag works to add this portion of the form only for `:new` (and not for :edit), but there ought to be a better way to do this.

Reply:
Yes... this has been bugging me too. It's in master but i couldn't think of a better way. Flags with various names are in almost all of the new/edit forms.

If you think of something better i'm all ears. At the least it would be good to standardize the name of the flag.

Reply:
Would something like the following work? (I don't have time to try it this week.)
Could the form `yield` back to the view? Then
- The :new view supplies the extra fields.
-  The :edit view just returns to the form.
See https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials-to-simplify-views

Reply:
I'll try it this week! If it works i'll apply it elsewhere.